fetch.bs

```diff
@@ -1485,21 +1495,29 @@ for each associated <a for="fetch group">fetch record</a> whose
 <var>origin</var> and <var>credentials</var>, run these steps:
 
 <ol>
- <li><p>If <a for=connection>connection pool</a> contains a
- <a>connection</a> whose <b>origin</b> is <var>origin</var> and
- <b>credentials</b> is <var>credentials</var>, return that
- <a>connection</a>.
+ <li><p>If <a for=connection>connection pool</a> contains a <a>connection</a> whose <b>origin</b> is
+ <var>origin</var> and <b>credentials</b> is <var>credentials</var>, return that <a>connection</a>.
+
+ <li><p>Let <var>connection</var> be null.
 
  <li>
-  <p>Let <var>connection</var> be the result of establishing an HTTP connection to
-  <var>origin</var>. [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]] [[!TLS]]
+  <p>Run these steps until fetch is <a for=fetch>terminated</a>:</p>
 
-  <p>If <var>credentials</var> is false, do <em>not</em> send a TLS client certificate.
+  <ol>
+   <li>
+    <p>Set <var>connection</var> to the result of establishing an HTTP connection to
+    <var>origin</var>. [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]] [[!TLS]]
```

Comment:
fwiw there are a few places where the line limit was already being ignored (or in some cases it was wrapping much sooner). If that was the case I've continued to ignore it rather than creating even more of a diff to look through.